README.md

```diff
@@ -133,9 +133,9 @@ The `DefaultsNoPii` and `DefaultsIncludingPii` renderers include default Compose
 don't need to do anything special. Additional Compose-specific renderers can be found in the
 `ComposableRenderers` object.
 
-To create a custom renderer for Compose, implement a `ViewStateRenderer` to handle values of type
+To create a custom renderer for Compose, implement a `ViewStateRenderer` to handle the values of type
 `ComposeView`. However, since Radiography gets most of its information about composables from their
-semantics properties, in most cases you shouldn't need to define any custom rendering logic. The
+semantics properties, in most cases you shouldn't need to define any custom rendering logic. 
```

Comment:
This might make sense

Reply:
I didn't use an article initially because this sentence is saying more than the ComposeView type has properties, but I think that's wrong because in the context of the rest of the paragraph, this sentence is talking about implementations of ViewRender, and in the context of a particular implementation, it makes more sense to talk about instance vs type. However, I think it's better to use a plural here to match above plural "values if type ComposeView", for consistency. Does that make sense?